spec/factories/claims.rb

```diff
@@ -17,7 +17,7 @@
       journey = Journeys.for_policy(evaluator.policy)
 
       begin
-        Journeys.for_policy(evaluator.policy)&.configuration.present?
+        raise ActiveRecord::RecordNotFound unless Journeys.for_policy(evaluator.policy)&.configuration.present?
```

Comment:
bug with the existing factory - it is possible to have `.present` return false - previously no exception was raised, so no journey_configuration gets created